### DISCUSSION TITLE
Sunbird Platform API Keys

### DISCUSSION THREAD

**Discussion by nagsides** [Sunbird-Ed/Community#440] [Installation]
Hello,
I am trying to install the Sunbird Platform on a VM (Ubuntu 20.04 LTS). I went through the steps mentioned here:
http://docs.sunbird.org/1.9/developer-docs/installation/developer_installation/

I followed all the steps and everything went smoothly, I went to localhost:3000 and displays a landing page with About, Contact, and Login links.

The above link also mentions:
"API Keys The Sunbird developer instance is powered by cloud-hosted Sunbird APIs, which require an API key. To get an API key, submit an API Key Request. In the form, provide information about your team and what brings you to Sunbird."

At present, I do not have the API Keys. How can I get it so that I can see the full functionality?
How do I generate the API Keys myself? Do I have to deploy other sunbird components?

What if I want to deploy it on the server? Can I deploy it on say 1 VM/2 VMs? 

Thank you
Nagesh

**Comment by Krishnaj20** [comment]
@chitranshu-keshav @gandham-santhosh @vinukumar-vs - FYA please.

**Comment by vinukumar-vs** [comment]
@nagsides Thanks for reaching us. 

We just want to understand what are you trying to do? 
Can you please explain in few lines/bullet points what are you trying to do or achieve.

Today for local testing of portal requires infra to be setup with all services. You can connect local portal to the infra to verify any issues with functional flows or you can modify as per the requirements.

@gandham-santhosh @Krishnaj20

**Reply by nagsides** [reply]
@vinukumar-vs Thanks for the reply.  I was trying to experience what Sunbird is and what its capabilities are. You mention that an infra need to be setup with all the services up. How do I go about setting up a full-blown Sunbird platform? We have a Mini cloud (all Ubuntu VMs) in place on the premises.  Could you please point me to the correct resource for the installation steps? Thanks.
@gandham-santhosh @Krishnaj20

**Reply by chitranshu-keshav** [reply]
@vinukumar-vs @rajeevsathish @gandham-santhosh Can you please get back on the above

**Comment by ajgemini** [comment]
> Hello, I am trying to install the Sunbird Platform on a VM (Ubuntu 20.04 LTS). I went through the steps mentioned here: http://docs.sunbird.org/1.9/developer-docs/installation/developer_installation/
> 
> I followed all the steps and everything went smoothly, I went to localhost:3000 and displays a landing page with About, Contact, and Login links.
> 
> The above link also mentions: "API Keys The Sunbird developer instance is powered by cloud-hosted Sunbird APIs, which require an API key. To get an API key, submit an API Key Request. In the form, provide information about your team and what brings you to Sunbird."
> 
> At present, I do not have the API Keys. How can I get it so that I can see the full functionality? How do I generate the API Keys myself? Do I have to deploy other sunbird components?
> 
> What if I want to deploy it on the server? Can I deploy it on say 1 VM/2 VMs?
> 
> Thank you Nagesh

@vinukumar-vs I have the same request. 
Out team is trying to build an e-learning platform  and for that we are trying to use Sunbird-Ed. I am trying to find the above as well. @nagsides  Could you please point out where the form links to access API can be found?
